core/config.js

```diff
@@ -49,7 +49,7 @@ Object.assign(config, {
         val = (type && type(val)) || val;
       }
     }
-    return val || fallback;
+    return val != null ? val : fallback;
```

Comment:
Why is this change relevant?
If `val` is:
- `null`, it will return `fallback` either way.
- `undefined`, it will return `fallback` either way.
- `0`, it will return `fallback` the old way (with the `||`), but return `0` the new way (with the `?`).